Sources/_CollectionsUtilities/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#[[
+This source file is part of the Swift Collections Open Source Project
+
+Copyright (c) 2022 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(_CollectionsUtilities
+  "Array+WithContiguousStorage Compatibility.swift"
+  "CMakeLists.txt"
```

Comment:
This should be removed, this is going to cause a build failure.

Reply:
Oh good eye -- I wonder how my Ninja test run didn't complain.

Reply:
Good question - not sure about that.  There must've been something which prevented that.

Reply:
Fixed in https://github.com/apple/swift-collections/pull/203. (Btw, apologies for merging this too soon -- I was trigger happy to get an upcoming merge to `main` go quickly.)